tox.ini

```diff
@@ -1,7 +1,16 @@
 [tox]
 envlist = py37,py39,py310,py311,py312,py313,pypy3
+
 [testenv]
-changedir = .tox
```

Comment:
Was this line really needed? If so, I'll have to update the `[testenv:mypy]` to not use that working directory or to explicitly pass the mypy config file to the mypy command (mypy 1.15 added support to walk up directory trees to find `mypy.ini`, before that we need to explicitly pass the config file location if it's not in the working directory)